src/libraries/System.Reflection/tests/TypeInfoTests.cs

```diff
@@ -1474,7 +1474,6 @@ public void IsVisible(Type type, bool expected)
         [InlineData(typeof(int), "System")]
         [InlineData(typeof(TI_BaseClass[]), "System.Reflection.Tests")]
         [InlineData(typeof(TI_BaseClass.PublicNestedClass1[]), "System.Reflection.Tests")]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/42633", TestRuntimes.Mono)]
```

Comment:
Should we add another `InlineData` with `PublicNestedClass1[][]` and also some multiply-nested class?

Also looks like the Mono could would also return the enclosing namespace for pointer types.  So maybe also InlineData for `PublicNestedClass1*` ?

Reply:
We could definitely add `PublicNestedClass1[][]` as `typeof(T[])==typeof(T)`. But does that also hold for pointers standardwise?

Reply:
> But does that also hold for pointers standardwise?

No idea.  But if we add a test, we can record whatever the behavior happens to be.  (in particular I want to confirm Mono and CoreCLR do the same thing for pointer types)

Reply:
Added the test cases. There are no apparent differences in how Mono and CoreCLR handle this.